source/agent/quic/index.js

```diff
@@ -198,14 +191,16 @@ module.exports = function (rpcClient, selfRpcId, parentRpcId, clusterWorkerIP) {
             log.error('Create connection failed', connectionId, connectionType);
             return callback('callback', {type: 'failed', reason: 'Create Connection failed'});
         }
-        connections.addConnection(connectionId, connectionType, options.controller, conn, 'in')
-        .then(onSuccess(callback), onError(callback));
+        conn.bindRouterAsSourceCallback = function(stream) {
+          router.addLocalSource(connectionId, connectionType, stream);
+        }
+        onSuccess(callback)();
     };
 
     that.unpublish = function (connectionId, callback) {
         log.debug('unpublish, connectionId:', connectionId);
-        var conn = connections.getConnection(connectionId);
-        connections.removeConnection(connectionId).then(function(ok) {
+        var conn = router.getConnection(connectionId);
+        router.removeConnection(connectionId).then(function(ok) {
             if (conn && conn.type === 'internal') {
                 internalConnFactory.destroy(connectionId, 'in');
```

Comment:
Use `conn.close` for both 'internal' or 'quic' here

Reply:
Fixed.